control-plane-operator/controllers/hostedcontrolplane/kas/kms/ibmcloud.go

```diff
@@ -60,12 +59,6 @@ func NewIBMCloudKMSProvider(ibmCloud *hyperv1.IBMCloudKMSSpec, kmsImage string)
 }
 
 func (p *ibmCloudKMSProvider) GenerateKMSEncryptionConfig() (*v1.EncryptionConfiguration, error) {
-	keyVersionKeyEntryMap := buildIBMCloudKeyVersionKeyEntryMap(p.ibmCloud.KeyList)
```

Comment:
Can the `buildIBMCloudKeyVersionKeyEntryMap` func (as well as `buildIBMCloudKMSInfoEnvVar`) be removed entirely? Are they still useful with KMS v1 removed?

Reply:
I believe this is still needed. The key map is eventually passed to the KMS container.

https://github.com/openshift/hypershift/blob/2b6f01a6f93c6dd79f265848bd3005e7421a969c/control-plane-operator/controllers/hostedcontrolplane/kas/kms/ibmcloud.go#L245